plugins/jdg-nightly/README.txt

```diff
@@ -0,0 +1,8 @@
+Data Grid nightly build RadarGun Plug-in
+--------------------------------------
+
+This plugin is used to host the nightly Data Grid builds.
+
+after that you need to rebuild RadarGun with
```

Comment:
Is there something missing on this line? It starts with lower case.

Reply:
Hm, yeah this looks weird